tracer/test/benchmarks/Benchmarks.Trace/AppSecBodyBenchmark.cs

```diff
@@ -45,13 +45,13 @@ public class AppSecBodyBenchmark
         static AppSecBodyBenchmark()
         {
             var dir = Directory.GetCurrentDirectory();
-            while (!Directory.GetDirectories(dir).Any(s=>s.Contains("shared")))
+            while (!Directory.GetDirectories(dir).Any(s => s.Contains("shared")))
             {
                 dir = Directory.GetParent(dir).FullName;
             }
 
             Environment.SetEnvironmentVariable("DD_APPSEC_ENABLED", "true");
-            var path = Path.Combine(dir, "bin", "monitoring-home", RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? $"win-{(Environment.Is64BitOperatingSystem ? "x64" : "x86")}" : string.Empty);
+            var path = Path.Combine(dir, "shared", "bin", "monitoring-home", RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? $"win-{(Environment.Is64BitOperatingSystem ? "x64" : "x86")}" : string.Empty);
```

Comment:
Small suggestion - you could have Nuke set the `DD_DOTNET_TRACER_HOME` env var automatically, as it knows exactly where this is built to - and then only override the value if it's missing?

Reply:
thanks great idea! done in last commit, the benchmarks seem to be fixed now. we still have a warning like native.dll was not found but it's normal, it s just that there's no clr profiler loaded of course, but the waf is loaded and invoked fine